.github/workflows/auto-close-github-discussions.yml

```diff
@@ -17,6 +17,12 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v4
+        with:
+          version: 10
```

Comment:
```suggestion
```
No need to specify the version here, there's a `packageManager` entry in `package.json` so it should download the right version, and, fwiw, we use pnpm 9 and not 10.

Reply:
Thanks for letting me know. Removed the version.